### PR TITLE
Dbic update

### PR DESCRIPTION
**Pull request by kemsguy7**
@manwar , this is an update to issue https://github.com/ChallengeForge/TheWeeklyChallenge/pull/9 as you requested, kindly review.

### CODE REVIEW

**Comment by manwar** [comment]
@kemsguy7 good job and clean code.
Can you please follow the same pattern and get other tables done too?

You can find the table details in the following SQL file.

https://github.com/ChallengeForge/TheWeeklyChallenge/blob/main/database/build_schema.sql

**Comment by kemsguy7** [comment]
Alright  @manwar , I'll get that done

**Comment by kemsguy7** [comment]
@manwar, I've writtenn the dbic to include all other tables as you requested . Please kindly review

**Comment by manwar** [comment]
@kemsguy7 just checking, does this cover all the tables in the schema?

**Comment by kemsguy7** [comment]
> @kemsguy7 just checking, does this cover all the tables in the schema?

Yes @manwar it covers all tables

**Comment by kemsguy7** [comment]
@manwar , does everything look good? do you need any other change on this issue?

File: lib/Models/PWC/Champions.pm [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package PWC::Schema::Result::Champion;
```
Comment (manwar) [inline lib/Models/PWC/Champions.pm]: Package name should match folder path. 

I suggest name your package like:

      package Models::PWC::Schema::Result::Champion;

and it should be saved in the folder:
     
      lib/Models/PWC/Schema/Result/Champion.pm

**Review by manwar — COMMENTED** [review]
As suggested in the comment, apply the same to all.

Also keep in mind we are going to use Dancer2::Plugin::DBIC

        https://metacpan.org/pod/Dancer2::Plugin::DBIC

I casually read the documentation it mentioned something about generating schemas, please read and see if it is useful for us to automate.

        https://metacpan.org/pod/Dancer2::Plugin::DBIC#SCHEMA-GENERATION

**Comment by kemsguy7** [comment]
Okay mentor @manwar,, i'm applying the requested changes now

File: lib/Models/PWC/Champions.pm (hunk 1 above)
Comment (kemsguy7) [reply lib/Models/PWC/Champions.pm]: @manwar , please should I 
do this for all the tables?

**Comment by kemsguy7** [comment]
@manwar , i've pushed some more updates, kindly review

**Comment by manwar** [comment]
@kemsguy7 Looks like you are adding generated files too.
We don't need MYMETA.* files.
Also don't include blib/* folder.

The lib/* is important and only needed.

**Comment by kemsguy7** [comment]
@manwar, i've added the update as requested. Kindly review

**Comment by kemsguy7** [comment]
@manwar in this last commit, i have been able to automatically generate the schema using the dbicdump tool, but I had to covert the database to sqlite. Please let me know if that works or we should just stick to the previous MySQL version?

**Comment by manwar** [comment]
Thanks @kemsguy7 for the contributions.

**Pull request merged**